modules/benchmark/R/create.BRR.R

```diff
@@ -9,48 +9,49 @@
 ##' 
 ##' @author Betsy Cowdery 
 
-create.BRR <- function(ensemble.id, workflow, con) {
+create.BRR <- function(ens_wf, con){
```

Comment:
changed argument to something hard to understand, but didn't update the documentation (param still listed as ensemble.id)